stratum/hal/lib/tdi/tdi_constants.h

```diff
@@ -15,7 +15,10 @@ namespace tdi {
 // TNA Extern types
 constexpr uint32 kTnaExternActionProfileId = 131;
 constexpr uint32 kTnaExternActionSelectorId = 132;
-constexpr uint32 kTnaExternDirectCounter = 133;
+constexpr uint32 kTnaExternDirectCounter = 130;
+constexpr uint32 kTnaExternPacketModMeter = 133;
+constexpr uint32 kTnaExternDirectPacketModMeter = 134;
```

Comment:
These are not TNA externs. They are ES2K externs

Reply:
Fixed